lib_comfyui/polling_client.py

```diff
@@ -133,6 +135,16 @@ async def set_polling_server_focused_webui_client_id(request):
         print(f'[sd-webui-comfyui] set client webui_client_id - \n{webui_client_id}')
         return web.json_response()
 
+    @instance.routes.get("/sd-webui-comfyui/default_workflow")
+    async def get_default_workflow(request):

```

Comment:
this function should be defined in its own file, most likely. the hijack logic in this file could be moved somewhere else?